server/src/internalClusterTest/java/org/elasticsearch/indices/memory/breaker/HierarchyCircuitBreakerTelemetryIT.java

```diff
@@ -142,37 +97,29 @@ public void testCircuitBreakerTripCountMetric() {
         fail("Expected exception not thrown");
     }
 
-    private List<Measurement> getMeasurements(String dataNodeName) {
-        final TestTelemetryPlugin dataNodeTelemetryPlugin = internalCluster().getInstance(PluginsService.class, dataNodeName)
-            .filterPlugins(TestCircuitBreakerTelemetryPlugin.class)
+    @After
+    public void resetClusterSetting() {
```

Comment:
Fix: we need to reset cluster settings after the tests, before the global IT tests cleanup; having a very strict circuit breaker during global IT tests cleanup will trip it and break the tests